docs/test-IM/test-eng.html

```diff
@@ -1,227 +1,225 @@
+			</div>
+			<div class="brand">
+				<div class="container">
+					<div class="row ">
+						<nav class="col-md-9 col-lg-10 ftr-urlt-lnk">
+							<h2 class="wb-inv">About this site</h2>
+							<ul>
+								<li><a href="https://www.canada.ca/en/social.html">Social media</a></li>
+								<li><a href="https://www.canada.ca/en/mobile.html">Mobile applications</a></li>
+								<li><a href="https://www1.canada.ca/en/newsite.html">About Canada.ca</a></li>
+								<li><a href="https://www.canada.ca/en/transparency/terms.html">Terms and conditions</a></li>
+								<li><a href="https://www.canada.ca/en/transparency/privacy.html">Privacy</a></li>
+							</ul>
+						</nav>
+						<div class="col-xs-6 visible-sm visible-xs tofpg">
+							<a href="#wb-cont">Top of Page <span class="glyphicon glyphicon-chevron-up"></span></a>
+						</div>
+						<div class="col-xs-6 col-md-3 col-lg-2 text-right">
+							<img src="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/assets/wmms-blk.svg"
+								alt="Symbol of the Government of Canada" />
+						</div>
 					</div>
 				</div>
 			</div>
-		</div>
-	</footer>
+		</footer>
 
-	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"
-		integrity="sha384-rY/jv8mMhqDabXSo+UCggqKtdmBfd3qC2/KvyTDNQ6PcUJXaxK1tMepoQda4g5vB"
-		crossorigin="anonymous"></script>
-	<script src="https://wet-boew.github.io/themes-dist/GCWeb/wet-boew/js/wet-boew.min.js"></script>
-	<script src="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/js/theme.min.js"></script>
+		<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.2.4/jquery.min.js"
+			integrity="sha384-rY/jv8mMhqDabXSo+UCggqKtdmBfd3qC2/KvyTDNQ6PcUJXaxK1tMepoQda4g5vB"
+			crossorigin="anonymous"></script>
+		<script src="https://wet-boew.github.io/themes-dist/GCWeb/wet-boew/js/wet-boew.min.js"></script>
+		<script src="https://wet-boew.github.io/themes-dist/GCWeb/GCWeb/js/theme.min.js"></script>
 
-	<!-- used to check for the path of config.JSON file. 
-		If not declared, config.JSON is expected to be in the same directory as InvitationManger.js -->
-	<script>
-		window.imConfigPath = "https://servicecanada.github.io/invitation-manager/docs/";
-	</script>
+		<!-- used to check for the path of config.JSON file. 
+			If not declared, config.JSON is expected to be in the same directory as InvitationManger.js -->
+		<script>
+			window.imConfigPath = "https://servicecanada.github.io/invitation-manager/docs/";
+		</script>
 
-	<!-- Invitation Manager JS Files -->
-	<script src="https://servicecanada.github.io/invitation-manager/src/Overlay.js"></script>
-	<script src="https://servicecanada.github.io/invitation-manager/src/InvitationManager.js"></script>
-	<!-- End Invitation Manager JS Files-->
-</body>
+		<!-- Invitation Manager JS Files -->
+		<script src="https://servicecanada.github.io/invitation-manager/src/Overlay.js"></script>
+		<script src="https://servicecanada.github.io/invitation-manager/src/InvitationManager-eric.js"></script>
```

Comment:
This still has `-eric` in the link:
```suggestion
		<script src="https://servicecanada.github.io/invitation-manager/src/InvitationManager.js"></script>
```